### PR TITLE
Autoescape layout helpers

### PR DESCRIPTION
**Pull request by jodosha**
@lotus/core-team

### CODE REVIEW

File: test/fixtures.rb [hunk 1]
```diff
@@ -339,7 +339,7 @@ class StoreLayout
       include Store::Helpers::AssetTagHelpers
 
       def head
-        %(<meta name="lotusrb-version" content="0.3.1">)
+        Lotus::Utils::Escape::SafeString.new %(<meta name="lotusrb-version" content="0.3.1">)
```
Comment (AlfonsoUceda) [inline test/fixtures.rb]: notice lotus version 0.3.1 in this tag, do you think this tag is useful or secure? I mean, you're telling the world what lotus version you are using.

File: lib/lotus/view/rendering/layout_scope.rb [hunk 2]
```diff
@@ -206,7 +208,7 @@ def method_missing(m, *args, &blk)
           elsif layout.respond_to?(m)
             layout.__send__(m, *args, &blk)
           else
-            super
+            ::Lotus::View::Escape.html(super)
```
Comment (AlfonsoUceda) [inline lib/lotus/view/rendering/layout_scope.rb]: do we need tests for this?

**Comment by joneslee85** [comment]
LGTM :+1:

File: lib/lotus/view/rendering/layout_scope.rb (hunk 2 above)
Comment (jodosha) [inline lib/lotus/view/rendering/layout_scope.rb]: @AlfonsoUceda Without it, existing tests won't pass.

File: test/fixtures.rb (hunk 1 above)
Comment (jodosha) [inline test/fixtures.rb]: @AlfonsoUceda I don't understand, this is only an example of a `<meta>` HTML tag. It isn't used by Lotus apps.

The fixture could've been something like this:

```ruby
Lotus::Utils::Escape::SafeString.new %(<meta name="foo" content="bar">)
```

:wink:

**Pull request merged**